### PR TITLE
Vince Carollo

### PR DESCRIPTION
**Pull request by VinceCarollo**

### CODE REVIEW

File: lib/pantry.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  def restock(ingredient, qty)
+    if !stock.keys.include?(ingredient)
+      stock[ingredient] = qty
+    else
+      stock[ingredient] += qty
+    end
+  end
+
+  def stock_check(ingredient_needed)
+    @stock.sum do |ingredient, qty|
```
Comment (ameseee) [inline lib/pantry.rb]: You can probably also just return `@stock[ingredient_needed]`!

File: lib/pantry.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      qty if ingredient == ingredient_needed
+    end
+  end
+
+  def enough_ingredients_for?(recipe)
+    enough_ingredients = false
```
Comment (ameseee) [inline lib/pantry.rb]: Nice job powering through this, I think you only needed one iteration - an `all?` to handle this, so may want to take a little time to refactor that for readability/simplicity.

**Comment by ameseee** [comment]
Hi @VinceCarollo, nice work! I left a couple comments where you have some verbose solutions that feel a bit over-engineered, may want to take a look. Overall, looks solid though!!

File: lib/pantry.rb (hunk 2 above)
Comment (VinceCarollo) [reply lib/pantry.rb]: Yea I made this one way too long and spent way too much time on it  😂.  Over done for sure. If I would've figured this out earlier I might've been able to finish iteration 5. Oh well. Thanks for the feedback!

**Pull request closed without merging**